packages/gatsby/src/redux/__tests__/jobsv2.js

```diff
@@ -90,7 +90,20 @@ describe(`Job v2 actions/reducer`, () => {
       inputPaths: [],
     })
     expect(jobsManager.removeInProgressJob).toHaveBeenCalledTimes(1)
-    expect(jobsManager.enqueueJob).toMatchSnapshot()
+    expect(jobsManager.enqueueJob).toHaveBeenCalledWith({
```

Comment:
This test was failing because it was snapshotting a Promise object, and using AsyncLocalStorage in other tests meant that there were added internal fields on global Promise. The object it's snapshotting is a jest mock, so it makes more sense to use the actual mock assertions instead of snapshotting its internal state